bitrise.yml

```diff
@@ -1,31 +1,88 @@
-format_version: 5
+format_version: 8
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 app:
   envs:
-  - RELEASE_VERSION: 2.2.3
-  - STEP_ID_IN_STEPLIB: fabric-crashlytics-beta-deploy
+  - SAMPLE_APP_URL: https://github.com/bitrise-samples/iOS-Sample.git
+  - SAMPLE_APP_BRANCH: TOOL-248-firebase
 
   # define these in your .bitrise.secrets.yml
-  - BITRISE_IPA_PATH: $BITRISE_IPA_PATH
-  - BITRISE_DSYM_PATH: $BITRISE_DSYM_PATH
   - CRASHLYTICS_API_KEY: $CRASHLYTICS_API_KEY
   - CRASHLYTICS_BUILD_SECRET: $CRASHLYTICS_BUILD_SECRET
 
 workflows:
-  # ----------------------------------------------------------------
-  # --- workflow to test this step
   test:
     before_run:
     - audit-this-step
+    - checkout
+    # - test_fabric
```

Comment:
Why is the `test_fabric` workflow disabled?

Reply:
fixed